api/src/main/java/io/opentelemetry/distributedcontext/DistributedContext.java

```diff
@@ -57,9 +56,7 @@
    */
   interface Builder {
     /**
-     * Sets the parent {@code DistributedContext} to use. If not set, the value of {@code
-     * Entryger.getCurrentDistributedContext()} at {@link #build()} or {@link #buildScoped()} time
```

Comment:
Why remove this line?

Reply:
Correct - we can keep this line, and simply remove the `buildScoped()` mention.

Reply:
I could not find any definition on what an `Entryger` is supposed to be nor any method called `getCurrentDistributedContext` in the whole repo. I also checked `DistributedContextSdk.Builder` and saw that it defaults to `parent=null` and doesn't acquire any implicit one.
Therefore I thought these were some leftovers which were forgotten to be removed.
Are you sure the original statement was correct?

Reply:
`Parent` is used here: https://github.com/open-telemetry/opentelemetry-java/blob/50a2db42b257cb9618131bffc2fbc997785b7f38/sdk/src/main/java/io/opentelemetry/sdk/distributedcontext/DistributedContextSdk.java#L60.

Reply:
Yes it is, but, unless explicitly specified, parent stays null. Therefore the statement
> `     * Sets the parent {@code DistributedContext} to use. If not set, the value of {@code
     * Entryger.getCurrentDistributedContext()} at {@link #build()} or {@link #buildScoped()} time
     * will be used as parent.`

is incorrect, right?

Reply:
`Entryger` seems to be a typo, it should be `DistributedConetxtManager. getCurrentDistributedContext()`.   

> DistributedContextSdk.Builder and saw that it defaults to parent=null and doesn't acquire any implicit one.

This looks like a bug. Instead of null, default value should be the value of `DistributedConetxtManager. getCurrentDistributedContext()`. You can leave a TODO here to fix it.

Reply:
Makes sense, done!